solutions/account-infrastructure-base/variables.tf

```diff
@@ -45,37 +38,21 @@ variable "prefix" {
   }
 }
 
-variable "kms_key_crn" {
-  type        = string
-  description = "The CRN of the key management service key to encrypt the Object Storage bucket. Required if `provision_activity_tracker_cos` is true."
-  default     = null
-}
-
-variable "cos_bucket_management_endpoint_type" {
-  description = "The type of endpoint for the IBM terraform provider to use to manage the bucket. (public, private or direct)"
-  type        = string
-  default     = "public"
-  validation {
-    condition     = contains(["public", "private", "direct"], var.cos_bucket_management_endpoint_type)
-    error_message = "The specified management_endpoint_type_for_bucket is not a valid selection."
-  }
-}
-
 variable "allowed_ip_addresses" {
   description = " A list of the IP addresses and subnets from which IAM tokens can be created for the account."
   type        = list(string)
   default     = []
 }
 
-variable "provision_activity_tracker_cos" {
-  type        = bool
-  description = "Whether to enable creating an Activity Tracker route, Object Storage instance, and bucket."
-  default     = false
+variable "global_resource_group_name" {
+  type        = string
+  description = "The name of the global resource group to create, takes precedence over all other resource group name variables. If `prefix` is provided, it is prefixed on the name in the following format: `<prefix>-<global_resource_group_name>`."
```

Comment:
Can we make it clear that when a value is set for this, no other resource groups are created